### ISSUE TITLE
Device naming hotspot weirdness

### ISSUE THREAD

**Issue by andrew-axiom** [ColquhounAudio/AxiomBuild#17]
When I originally booted up the new sesame image, my device came up as Axiomair-85xxxx. I tested the hotspot fallback and it work correctly, but the hostpot appeared as a different device name Axiomair-3Axxxx instead of the name I changed it to, AW1. Not sure how this is happening and why the -xxxxxx is not consistent. It would be very confusing to a customer with more than one unit. The device should be whatever it is names as the hotspot name...in my case it should have appeared as AW1.

**Comment by IanColquhoun** [comment]
I had the same problem.  When I took it home last night I put it on my home network using WAC.  This worked but I got the error message this time.  When I came back to the office this morning it was back in Hotspot mode but no flashing red light, just the blue one.  I had renamed it IanOffice yesterday but in Hotspot mode it always shows as AxiomAir-xxxxxx; everywhere else it is IanOffice.  The .xxxxxx changed from yesterday when it was 670f7c to 370e2a today.
